src/client/app/components/HeaderComponent.jsx

```diff
@@ -22,16 +22,21 @@ export default function HeaderComponent(props) {
 	};
 	const loginLinkStyle = {
 		// Displays the login button link only if the user is not logged in or is explicitly told to display by the parent component
-		display: localStorage.getItem('token') || props.renderLoginButton === 'false' ? 'none' : 'inline'
+		display: localStorage.getItem('token') || props.renderLoginButton === false ? 'none' : 'inline'
```

Comment:
This is a pure styling thing, which is used heavily in the Redux documentation. Combine d with the comment above, I think it is clear enough.